docs/contrib/workflow.md

```diff
@@ -0,0 +1,232 @@
+   #1234 but do not close it. For example when you develop some big feature you can
+   have several PRs with different parts of your feature. Only last PR must have tag
+   `Closes`. Rest of your PRs must use tag `Related to` to do not close GitHub issue
+   when they will be merged.
+
+The previous paragraph says that you can have several commit in your PR. Only the first one
+must contain full commit message with title, description and tags. Rest commits
+in your branch can have only meaningful title. But remember that the best practice is to
+have only one commit per branch/PR to do not face with the problem of meaningless
+commit titles at all.
+
+## Review
+
+To show your changes to the other contributors you must
+[create](https://github.com/telstra/open-kilda/compare) a GinHub pull request.
+
+### GitHub pull request
+
+GitHub requires two branches for a pull request: base branch and your branch.
+Base branch for most of the PRs is `develop` branch.
+But if you want to split your changes on several PRs you can choose some other branch
+as a base.
+
+#### Examples 1:
+
+A developer has branch `feature/ha_flows` with a new feature and wants to create a PR.
+He/she creates new GitHub PR from branch `feature/ha_flows` with the base branch `develop`.
+A QA has branch `tests/ha_flows_func_tests` with tests for developer's feature. He/she
+creates new PR from branch `tests/ha_flows_func_tests` with the base branch
+``feature/ha_flows``.
+
+#### Examples 2:
+
+A developer is working on some big feature. If he/she put all changes into one PR it can
+have several thousand lines of code. It will be very hard to review. It this case
+a developer should split his/her changes into several PRs:
+
+| PR title                                      | Base branch                 | Branch                      | Tags             |
+|-----------------------------------------------|-----------------------------|-----------------------------|------------------|
+| HA flows Part 1: Added API objects            | `develop`                   | `feature/ha_flow_api`       | Related to #1234 |
+| HA flows Part 2: Added messaging objects      | `feature/ha_flow_api`       | `feature/ha_flow_messaging` | Related to #1234 |
+| HA flows Part 3: Implemented create operation | `feature/ha_flow_messaging` | `feature/ha_flow_create`    | Related to #1234 |
+| HA flows Part 4: Implemented delete operation | `feature/ha_flow_create`    | `feature/ha_flow_delete`    | Closes #1234     |
+
+
+If your PR has more than 1000 lines of code - please try to split in into several PRs.
+It is not mandatory rule, but it is highly recommended. It is very hard to review
+1000+ lines of code.
+
+### Pull request labels
+Your PR must have labels. We have different label categories.
+1. PR type. One of three labels: `feature`, `bugfix`, `improvment`.
+   Pick one which describes your changes best.
+2. Area. `area/testing`, `area/docs`, etc. Describes area which you are changing.
```

Comment:
Should test PR inherit area from the tested feature and add it's own `area/testing` label or only use `area/testing` one?